### PR TITLE
add beginner friendly docs

### PR DESCRIPTION
**Pull request by 007vasy**
added some docs to make it easier for beginners to start, and pros to look at examples if they get stuck

### CODE REVIEW

**Comment by 007vasy** [comment]
@KuphJr added docs for beginners

**Comment by KuphJr** [comment]
Thanks for the PR!

We are doing some reworking to how users set up their environment this week, so I am going to hold off on merging until I can confirm that all the documentation is in alignment.

**Comment by 007vasy** [comment]
ping he if you want me to be a guinea pig

**Comment by KuphJr** [comment]
This branch will need to be rebased given the recent updates to the documentation.  @007vasy Can you update this branch & request another review?

File: README.md [hunk 1]
```diff
@@ -24,6 +26,20 @@
 <p>It works by using a <a href="https://chain.link/education/blockchain-oracles#decentralized-oracles">decentralized oracle network</a> (DON).<br>When a request is initiated, each node in the DON executes the user-provided JavaScript code simultaneously.  Then, nodes use the <a href="https://docs.chain.link/architecture-overview/off-chain-reporting/">Chainlink OCR</a> protocol to come to consensus on the results.  Finally, the median result is returned to the requesting contract via a callback function.</p>
 <p>Chainlink Functions also enables users to share encrypted secrets with each node in the DON.  This allows users to access APIs that require authentication, without exposing their API keys to the general public.
 
+# For BEGINNERS
+
+If you're new to web3, we recommend starting with the (Functions - Getting Started)[https://docs.chain.link/chainlink-functions/getting-started/] before diving into the code.
```
Comment (Rahul-Prasad-07) [inline README.md:31]: Need changes :  

replace  this --> (Functions - Getting Started)[https://docs.chain.link/chainlink-functions/getting-started/]  with well formatted link structure 

Add this --> [Functions - Getting Started](https://docs.chain.link/chainlink-functions/getting-started/])

File: README.md [hunk 2]
```diff
@@ -24,6 +26,20 @@
...
+
+The above document will help you:
+
+- Set up a wallet
+- Get founds
+- Provides more detailed step-by-step description of what you need to do to put data on chain
+
+## Tutorials & examples
+
+For more detailed tutorials and examples, check out the (Chainlink Functions Tutorials)[https://docs.chain.link/chainlink-functions/tutorials/] to get started.
```
Comment (Rahul-Prasad-07) [inline README.md:41]: Same Issue : Typo 

Need changes :

Replace this --> (Chainlink Functions Tutorials)[https://docs.chain.link/chainlink-functions/tutorials/] with well formatted link structure

Add this --> [Chainlink Functions Tutorials](https://docs.chain.link/chainlink-functions/tutorials/)

**Comment by 007vasy** [comment]
@KuphJr reabase done, thanks to @Rahul-Prasad-07 for spotting the errors

**Review by KuphJr — APPROVED** [review]

**Pull request merged**